tests/unit/Assert/RepositoryNameAssertTest.php

```diff
@@ -50,6 +50,7 @@ public function provideValidRepositoryNames() {
 	 */
 	public function testGivenValidValue_assertParameterIsValidRepositoryNamePasses( $value ) {
 		RepositoryNameAssert::assertParameterIsValidRepositoryName( $value, 'test' );
+		$this->assertTrue( true );
```

Comment:
Note while this is a valid and very common workaround, there a dedicated feature for situations like this: `$this->addToAssertionCount(1);`

Reply:
I thought there was but couldn't' find the doc for it at the time.
Will fix!

Reply:
Done in https://github.com/wmde/WikibaseDataModel/commit/a55334a301d09d57f5d378a72fbbd1335f7d290f